### ISSUE TITLE
compat: hello world example of express.js doesn't work

### ISSUE THREAD

**Issue by kt3k** [denoland/deno#12834]
The following official hello world example of express.js worked a few versions earlier, but now it doesn't work (the program exits immediately)
```js
const express = require('express')
const app = express()
const port = 3000

app.get('/', (req, res) => {
  res.send('Hello World!')
})

app.listen(port, () => {
  console.log(`Example app listening at http://localhost:${port}`)
})
```

If I add `setTimeout(() => {}, 0)` at the end of the program, it works correctly. I guess something's wrong about remaining ops detections.

tested with deno 1.16.2.

**Comment by lucacasonato** [comment]
cc @bnoordhuis. I'd say this is an std error?

**Comment by bnoordhuis** [comment]
This was fixed recently in 22dcf8223.
